lib/vmpooler/pool_manager.rb

```diff
@@ -555,6 +581,17 @@ def _check_pool(pool)
         end
       end
 
+      # MIGRATIONS
+      $redis.smembers('vmpooler__migrating__' + pool['name']).each do |vm|
+        if inventory[vm]
+          begin
+            migrate_vm(vm, pool['name'])
+          rescue
```

Comment:
I have added the context of the failure to the log message and cleaned up the logger string.